libmscore/edit.cpp

```diff
@@ -1954,6 +1954,8 @@ void Score::deleteItem(Element* el)
                               else
                                     mea->undoChangeProperty(Pid::MEASURE_NUMBER_MODE, static_cast<int>(MeasureNumberMode::AUTO));
                               break;
+                        case MeasureNumberMode::HIDE:
```

Comment:
@Marr11317 please check whether that was your intention with #5817

Reply:
Exactly thanks, in fact, this code is unreachable. This `switch` is reached when a user selects a measure number and presses delete. In this case, `MeasureNumberMode::HIDE` is not possible, since there wouldn't be a measure number to delete at all.

Reply:
I see. MinGW (and hence gcc) complains about an unhandled case here.